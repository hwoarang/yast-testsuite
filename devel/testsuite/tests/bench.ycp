{

include "testsuite.ycp";
include "devel/bench.ycp";

global define func1() ``{
    y2error("func1");
};

global define func2() ``{
    y2error("func2");
};

DUMP("BENCH");
TEST(``(BENCH(``(func1()),0)), [], nil);
TEST(``(BENCH(``(func1()),3)), [], nil);

DUMP("MULTIBENCH");
TEST(``(MULTIBENCH([],0)), [], nil);
TEST(``(MULTIBENCH([],3)), [], nil);
TEST(``(MULTIBENCH([``(func1()),``(func2())],3)), [], nil);

}
