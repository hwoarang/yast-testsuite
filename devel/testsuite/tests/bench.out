Dump	BENCH
Log	times=0
Return	0
Log	func1
Log	func1
Log	func1
Return	0.00000000000000
Dump	MULTIBENCH
Log	times=0
Return	0
Return	[]
Log	func1
Log	func1
Log	func1
Log	func2
Log	func2
Log	func2
Return	[0.00000000000000, 0.00000000000000]
